scapy/contrib/bfd.py

```diff
@@ -58,10 +143,12 @@ def mysummary(self):
         )
 
 
-for _bfd_port in [3784,          # single-hop BFD
-                  4784,          # multi-hop BFD
-                  6784,          # BFD for LAG a.k.a micro-BFD
-                  7784]:         # seamless BFD
+for _bfd_port in [
+    3784,  # single-hop BFD
+    4784,  # multi-hop BFD
+    6784,  # BFD for LAG a.k.a micro-BFD
+    7784,
+]:  # seamless BFD
```

Comment:
nit: the comment is misplaced

Reply:
Oh did not even noticed code formating tool touch this one...